evenOccurrences/evenOccurrences.js

```diff
@@ -10,6 +10,16 @@
  *    * console.log(onlyEven); //  4
  *    */
 
-const evenOccurence = (arr) => {
-  // Your code here.
-};
+ const evenOccurence = (arr) => {
+   // Your code here.
+   const occur = [];
+   arr.forEach((value, index) => {
+     occur.push([value, 0]);
+   });
+   for (let i = 0; i < occur.length; i++) for (let j = 0; j < arr.length; j++) if (occur[i][0] === arr[j]) occur[i][1] += 1;
```

Comment:
Please nest your for loops in blocks. I know you can do it on one line, but it's hard to read this way